packages/scandipwa/src/component/ProductCompareButton/ProductCompareButton.style.scss

```diff
@@ -10,13 +10,24 @@
  */
 
 :root {
-    --product-compare-button-icon-background: var(--primary-base-color);
-    --product-compare-button-grey-background: var(--secondary-base-color);
-    --product-compare-button-grey-color: var(--secondary-dark-color);
+    --product-compare-button-color: var(--primary-base-color);
+    --product-compare-button-dark-color: var(--primary-dark-color);
+    --product-compare-button-light-color: #{$white};
+    --product-compare-button-grey-color: #d8d5d5;
```

Comment:
What is this color? Where does it came from? :D

Reply:
This color is the wishlist button color in product card.
I should move it to a separate variable and use the same values on both wishlist and compare buttons.